python/notebooks/data_simulation/Untitled.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
Accidental commit or forgot to name the notebook? If the latter, maybe you could rename it for clarity.

Reply:
Ah yep, renamed it.